.travis.yml

```diff
@@ -7,11 +7,7 @@ rust:
 os:
   - linux
 script:
-  - cargo build
-  - cargo test
-  - cargo test --features log-events
-  - "if [ $TRAVIS_RUST_VERSION = nightly ]; then cargo test --features unstable; fi"
-  - cargo test
-  - "cd string-cache-codegen/ && cargo build && cd .."
-  - "cd examples/event-log/ && cargo build && cd ../.."
-  - "cd examples/summarize-events/ && cargo build && cd ../.."
+  - cargo test --all
+  - cargo test --features log-events --all
+  - "if [ $TRAVIS_RUST_VERSION = nightly ]; then cargo test --features unstable --all; fi"
```

Comment:
Building a workspace with features causes those features to be silently ignored. The --manifest-path argument is necessary to build with non-default features.